tests/conftest.py

```diff
@@ -1,32 +1,32 @@
-import sys
-import os
 
-absolute_path = os.path.dirname(os.path.abspath(__file__))
-relative_bitcoin_path = ('../bitcoin_client')
-absolute_bitcoin_client_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../')
-sys.path.append(os.path.join(absolute_path, relative_bitcoin_path))
+import sys  # noqa: E402
+import os  # noqa: E402
 
-import random
-from typing import Tuple
-
-from test_utils.fixtures import *
-from test_utils.authproxy import AuthServiceProxy, JSONRPCException
-from test_utils import segwit_addr
-
-import shutil
-import subprocess
-from time import sleep
-from decimal import Decimal
-from pathlib import Path
+absolute_path = os.path.dirname(os.path.abspath(__file__))  # noqa: E402
+relative_bitcoin_path = ('../bitcoin_client')  # noqa: E402
+absolute_bitcoin_client_path = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), '../')   # noqa: E402
+sys.path.append(os.path.join(absolute_path, relative_bitcoin_path))  # noqa: E402
```

Comment:
You pobably already know `pathlib.Path` and have your reason not to use it here, still I think it is interesting to consider it on cluttered lines like these.

The direct equivalent would be
```python
from pathlib import Path
absolute_path = Path(__file__).absolute().parent
relative_bitcoin_path = Path("../bitcoin_client")
absolute_bitcoin_client_path = Path(__file__).absolute().parent.parent
sys.path.append(str(absolute_path / relative_bitcoin_path)) 
```

but given the unused variable, it rather is:

```suggestion
from pathlib import Path
absolute_path = Path(__file__).absolute().parent  # noqa: E402
sys.path.append(str(absolute_path / "bitcoin_client"))  # noqa: E402
```

It also now works with (most) builtin libraries, such as shutil:

```python
    shutil.copy(Path(__file__).parent / "bitcoin.conf"), BITCOIN_DIRNAME)
    # or in your case
    shutil.copy(absolute_path / "bitcoin.conf", BITCOIN_DIRNAME)
```

Reply:
Thanks! Done in 540158f46590ffb52520285fe3a2731501d82798

Reply:
Reverted, somehow it broke the CI. Leaving it for a future PR.